Q/Qt6Base/build_tarballs.jl

```diff
@@ -28,31 +28,51 @@ commonoptions=" \
 -openssl-linked  -nomake examples -release \
 "
 
+commoncmakeoptions="-DCMAKE_PREFIX_PATH=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DQT_HOST_PATH=$host_prefix -DQT_FEATURE_openssl_linked=ON"
+
+export LD_LIBRARY_PATH=$host_libdir:$LD_LIBRARY_PATH
```

Comment:
Note: this line is needed to get the musl version of the `uic` tool to run, it is I think a general problem when running host tools that link dynamically to libraries from other JLLs (glib here), so it might be good to get this in by default at some point.

Reply:
Mind opening a PR: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/f2112cfb60e36b190e2d8cbd215c65c76ce00f66/src/Runner.jl#L911-L923?